.github/workflows/labels.yml

```diff
@@ -270,27 +261,22 @@ jobs:
             if (context.payload.pull_request) {
               await handleIssueAndPullRequest(context.payload.pull_request)
             } else {
-              const cutoff = new Date(await (async () => {
-                // Manually triggered via UI when updatedWithin is set. Will fallthrough to the last
-                // option if the updatedWithin parameter is set to 0, which is the default.
-                const updatedWithin = Number.parseInt(process.env.UPDATED_WITHIN, 10)
-                if (updatedWithin) return new Date().getTime() - updatedWithin * 60 * 60 * 1000
-
-                // Normally a scheduled run, but could be workflow_dispatch, see above. Go back as far
-                // as the last successful run of this workflow to make sure we are not leaving anyone
-                // behind on GHA failures.
-                // Defaults to go back 1 hour on the first run.
-                return (await github.rest.actions.listWorkflowRuns({
-                  ...context.repo,
-                  workflow_id: 'labels.yml',
-                  event: 'schedule',
-                  status: 'success',
-                  exclude_pull_requests: true
-                })).data.workflow_runs[0]?.created_at ?? new Date().getTime() - 1 * 60 * 60 * 1000
-              })())
+              const workflowData = (await github.rest.actions.listWorkflowRuns({
+                ...context.repo,
+                workflow_id: 'labels.yml',
+                event: 'schedule',
+                status: 'success',
+                exclude_pull_requests: true,
+                per_page: 1
+              })).data
+
+              // Go back as far as the last successful run of this workflow to make sure
+              // we are not leaving anyone behind on GHA failures.
+              // Defaults to go back 1 hour on the first run.
```

Comment:
Should this comment mention the edge-case where the results are shifted by a PR opening/closing, leading to potentially missing in-between PRs?